src/lib/crypto.pl

```diff
@@ -1,20 +1,20 @@
 /* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-   Written 2020, 2021, 2022 by Markus Triska (triska@metalevel.at)
+   Written 2020-2023 by Markus Triska (triska@metalevel.at)
    Part of Scryer Prolog.
 
-   Predicates for cryptographic applications.
+/** <module> Predicates for cryptographic applications.
```

Comment:
You don't need to add `<module>`. It may seem to work but that's because it is adding  a <module> HTML tag to the output... (and that should be fixed in DJota because is wrong)

Reply:
Thank you, I have now removed the `<module>` prefix in all pending pull requests, and I hope it works! Please let me know if you notice anything else.